client.bal

```diff
@@ -46,23 +47,22 @@ public client class Client {
     # Retrieves a list of all Amazon S3 buckets that the authenticated user of the request owns.
     # 
     # + return - If success, returns a list of Bucket record, else returns error
+    @display {label: "Get buckets"}
     remote function listBuckets() returns @tainted Bucket[]|error {
-        map<string> requestHeaders = {};
-        http:Request request = new;
-
-        requestHeaders[HOST] = self.amazonHost;
-        requestHeaders[X_AMZ_CONTENT_SHA256] = UNSIGNED_PAYLOAD;
+        map<string> requestHeaders = {
```

Comment:
As this code section is repeated, shall we move to a separate function and call?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/commit/02312d664879220bb3bfe2db0e8786ae7af97a06